src/components/EditableField/EditableField.utils.ts

```diff
@@ -5,9 +5,9 @@ import { find } from '../../utilities/arrays'
 export const COMPONENT_KEY = 'EditableField'
 
 export const ACTION_ICONS = {
-  delete: 'cross-medium',
-  link: 'new-window-large',
-  plus: 'plus-medium',
+  delete: 'cross-small',
+  link: 'new-window',
+  plus: 'plus-small',
   valueOption: 'chevron-down',
```

Comment:
There are constants for these defined in `icons-all.js` that are exported. Could we reuse these?

For example:

```
export const ACTION_ICONS = {
  delete: ICONS.crossMedium,
  ...
}

Reply:
I think `ICONS` is an an object with keys that tie to the svg:

`ICONS.folder = "<svg ... />"`

Reply:
I was hoping we could use this https://github.com/helpscout/hsds-react/blob/master/src/components/Icon/icons.all.js#L212 but, nevermind, it turns out the keys and values are in the reverse order that we'd need.